packages/formation-react/src/components/SortableTable/SortableTable.jsx

```diff
@@ -30,7 +30,8 @@ class SortableTable extends Component {
      */
```

Comment:
This file was included because the linter wouldn't let me add my files before fixing it :)